packages/amplify_flutter/pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  amplify_core_plugin_interface: ^0.0.1-dev.4
+  plugin_platform_interface: ^1.0.1
```

Comment:
Hm, is this a Flutter framework thing? Why this change?

Reply:
yes, this is because of combining the `amplify_core_plugin_interface` package in amplify_flutter. This is to enable native platform communication which is needed for `configure()`